megamek/src/megamek/common/loaders/MtfFile.java

```diff
@@ -505,107 +507,103 @@ public Entity getEntity() throws EntityLoadingException {
-                        mech.setTechLevel(TechConstants.T_CLAN_TW);
-                        break;
-                    case 3:
-                        mech.setTechLevel(TechConstants.T_CLAN_ADVANCED);
-                        break;
-                    case 4:
-                        mech.setTechLevel(TechConstants.T_CLAN_EXPERIMENTAL);
-                        break;
-                    case 5:
-                        mech.setTechLevel(TechConstants.T_CLAN_UNOFFICIAL);
-                        break;
-                    default:
-                        throw new EntityLoadingException("Unsupported tech level: " + rulesLevel.substring(12).trim());
-                }
-                break;
-            case "Mixed (IS Chassis)":
-                switch (Integer.parseInt(rulesLevel.substring(12).trim())) {
-                    case 2:
-                        mech.setTechLevel(TechConstants.T_IS_TW_NON_BOX);
-                        break;
-                    case 3:
-                        mech.setTechLevel(TechConstants.T_IS_ADVANCED);
-                        break;
-                    case 4:
-                        mech.setTechLevel(TechConstants.T_IS_EXPERIMENTAL);
-                        break;
-                    case 5:
-                        mech.setTechLevel(TechConstants.T_IS_UNOFFICIAL);
-                        break;
-                    default:
-                        throw new EntityLoadingException("Unsupported tech level: " + rulesLevel.substring(12).trim());
-                }
-                mech.setMixedTech(true);
-                break;
-            case "Mixed (Clan Chassis)":
-                switch (Integer.parseInt(rulesLevel.substring(12).trim())) {
-                    case 2:
-                        mech.setTechLevel(TechConstants.T_CLAN_TW);
-                        break;
-                    case 3:
-                        mech.setTechLevel(TechConstants.T_CLAN_ADVANCED);
-                        break;
-                    case 4:
-                        mech.setTechLevel(TechConstants.T_CLAN_EXPERIMENTAL);
-                        break;
-                    case 5:
-                        mech.setTechLevel(TechConstants.T_CLAN_UNOFFICIAL);
-                        break;
-                    default:
-                        throw new EntityLoadingException("Unsupported tech level: " + rulesLevel.substring(12).trim());
-                }
-                mech.setMixedTech(true);
-                break;
-            case "Mixed":
-                throw new EntityLoadingException("Unsupported tech base: \"Mixed\" is no longer allowed by itself.  You must specify \"Mixed (IS Chassis)\" or \"Mixed (Clan Chassis)\".");
-            default:
-                throw new EntityLoadingException("Unsupported tech base: " + techBase.substring(9).trim());
+        String techBase = this.techBase.substring(9).trim();
+        if (techBase.equalsIgnoreCase("Inner Sphere")) {
```

Comment:
We should probably move this into a constant

Reply:
Noted, just wanted this PR to be 'to-the-point' so as to make the review simple.